src/common/getters.ts

```diff
@@ -96,15 +107,21 @@ export function getRequestor(
  * @example
  *
  * ```
- * const modes = getAccessModes(gConsentGrant);
+ * const modes = getAccessModes(accessGrant);
  * ```
  *
  * @param vc The Access Grant/Request
  * @returns The access modes the grant recipient can exercise.
  */
 export function getAccessModes(
-  vc: AccessGrantGConsent | AccessRequestGConsent
+  vc: AccessGrantGConsent | AccessRequestGConsent | AccessGrantOdrl
 ): AccessModes {
+  if (isCredentialAccessGrantOdrl(vc)) {
+    const modes = vc.credentialSubject.permission
```

Comment:
Ditto here about if it can be nullable or not an array

Reply:
same as above